packages/cli/openapi-ir-to-fern/src/__test__/__snapshots__/streaming.test.ts.snap

```diff
@@ -0,0 +1,338 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`open api parser streaming simple 1`] = `
+{
+  "definitionFiles": {
+    "__package__.yml": {
+      "errors": {},
+      "service": {
+        "auth": false,
+        "base-path": "",
+        "endpoints": {
+          "user": {
+            "auth": false,
+            "docs": undefined,
+            "errors": [],
+            "examples": [
+              {
+                "headers": undefined,
+                "path-parameters": undefined,
+                "query-parameters": undefined,
+                "request": {
+                  "stream": false,
+                },
+                "response": {
+                  "body": {},
+                },
+              },
+            ],
+            "method": "POST",
+            "path": "/v1/users",
+            "request": {
+              "body": {
+                "properties": {
+                  "id": {
+                    "docs": "",
+                    "type": "optional<string>",
+                  },
+                  "stream": {
+                    "docs": undefined,
+                    "type": "optional<literal<false>>",
```

Comment:
This shouldn't be optional - the literal we synthesize from `x-fern-streaming` should be sent unconditionally.

Reply:
Done in https://github.com/fern-api/fern/pull/2251/commits/1b763b2f791a071504c9f9a7aa05a668f39400f8